abixen-platform-common/src/main/java/com/abixen/platform/core/model/ThemeBase.java

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright (c) 2010-present Abixen Systems. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.abixen.platform.core.model;
+
+
+public interface ThemeBase {
+
+    int THEME_TITLE_MIN_LENGTH = 1;
+    int THEME_TITLE_MAX_LENGTH = 50;
+
+    int THEME_NAME_MIN_LENGTH = 5;
```

Comment:
Unneeded, in the task description was only title field. But let's stay name field min 1 max 50 and remove title and  description